sql/updates/world/2015_XX_XX_XX_world.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `character_instance` ADD COLUMN `extend` TINYINT(1) UNSIGNED NOT NULL DEFAULT '0';
+ALTER TABLE `instance` ADD COLUMN `lastresettime` int(10) unsigned NOT NULL DEFAULT '0';
```

Comment:
Why not use Capital column name `Extend` or UpperCamelCase in the compound name `LastResetTime`?